stolon/templates/keeper-ro-service.yaml

```diff
@@ -3,11 +3,10 @@ kind: Service
 metadata:
   name: {{ template "stolon.keeper.fullname" . }}
   labels:
-    app: {{ template "stolon.keeper.fullname" . }}
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: "stolon-keeper"
```

Comment:
I suppose this should be `app: {{ template "stolon.keeper.fullname" . }}`

Reply:
Yes